src/fiatconnect/saga.ts

```diff
@@ -0,0 +1,52 @@
+import { call, put, select, spawn, takeLeading } from 'redux-saga/effects'
+import {
+  fiatConnectCashInEnabledSelector,
+  fiatConnectCashOutEnabledSelector,
+} from 'src/app/selectors'
+import { fetchQuotes, FiatConnectQuoteError, FiatConnectQuoteSuccess } from 'src/fiatconnect'
+import {
+  fetchFiatConnectQuotes,
+  fetchFiatConnectQuotesCompleted,
+  fetchFiatConnectQuotesFailed,
+} from 'src/fiatconnect/slice'
+import { LocalCurrencyCode } from 'src/localCurrency/consts'
+import { getLocalCurrencyCode } from 'src/localCurrency/selectors'
+import { UserLocationData } from 'src/networkInfo/saga'
+import { userLocationDataSelector } from 'src/networkInfo/selectors'
+import { currentAccountSelector } from 'src/web3/selectors'
+
+export function* handleFetchFiatConnectQuotes({
+  payload: params,
+}: ReturnType<typeof fetchFiatConnectQuotes>) {
+  const { flow, digitalAsset, cryptoAmount } = params
+  const userLocation: UserLocationData = yield select(userLocationDataSelector)
+  const account: string | null = yield select(currentAccountSelector)
+  const localCurrency: LocalCurrencyCode = yield select(getLocalCurrencyCode)
+  const fiatConnectCashInEnabled: boolean = yield select(fiatConnectCashInEnabledSelector)
+  const fiatConnectCashOutEnabled: boolean = yield select(fiatConnectCashOutEnabledSelector)
+
+  try {
+    // @ts-ignore TS complains about 'No overload matches this call' and I can't figure it out
```

Comment:
I think this is because of the type of `account`. Its `string | null` in this function vs `string` in FetchQuotesInput: https://github.com/valora-inc/wallet/pull/2674/files#diff-6a81d291c56147f95756fd7bd75460398eba135ed37571281b8345270c2cb65fL164